docarray/typing/url/any_url.py

```diff
@@ -35,6 +38,31 @@ def _to_node_protobuf(self) -> 'NodeProto':
 
         return NodeProto(text=str(self), type=self._proto_type_name)
 
+    @classmethod
+    def validate(
+        cls: Type[T],
+        value: Union[T, np.ndarray, Any],
+        field: 'ModelField',
+        config: 'BaseConfig',
+    ) -> T:
+        import os
+
+        try:
+            if not value.startswith('http') and not os.path.isabs(value):  # type: ignore
+                is_relative = True
+                value = os.path.abspath(value)  # type: ignore
```

Comment:
I think for stuff like this we should to the proper thing and assign to a new variable instead of silencing mypy